resources/chefdk/msi/localization-en-us.wxl.erb

```diff
@@ -20,4 +20,9 @@
   <String Id="FeaturePSModuleName"><%= friendly_name %> PowerShell wrappers</String>
   <String Id="FeatureChefDkEnvHacks">ChefDK Environment Customizations</String>
   <String Id="FeatureChefDkEnvHacksDesc">Allows ChefDK to try to find a suitable HOME drive when running Ruby programs.</String>
+  
+  <String Id="ChefDkStartMenuShortcutDefName"><%= friendly_name %></String>
+  <String Id="ChefDkStartMenuShortcutDefDescription"><%= friendly_name %> Command Prompt</String>
```

Comment:
Where does this show up -- in the installer UX? Wouldn't something like "Start Menu shortcut" be more appropriate in the installer?